### DISCUSSION TITLE
Multiple user account support

### DISCUSSION THREAD

**Discussion by FillingTheVoidOnYT** [syphon-org/syphon#726] [Ideas]
Allow the app to be able to have multiple users connected to one app from different instances. Basically multiple accounts

**Comment by 0x1a8510f2** [comment]
Syphon already supports multi account - we were one of the first to do so.

![Screenshot_20220916-111524~2.png](https://user-images.githubusercontent.com/83190128/190617469-759ca20f-9eac-410b-bf8d-9e9ef95f512d.png)

(Click on the 3 dots next to your account).

**Comment by notramo** [comment]
I guess this discussion indicates that the current design is not straightforward enough.

**Reply by EdGeraghty** [reply]
Indeed - the multiple account flow is currently under a full redesign.
